.github/workflows/test.yml

```diff
@@ -1,6 +1,9 @@
 name: e2e-tests
 
-on: push
+on:
+  push:
+    branches:
+      - master
```

Comment:
Please remember to restore to the original version
just kindly reminder before merge*